docs/winnt/cross-build-for-mingw64.md

```diff
@@ -0,0 +1,184 @@
+# Cross-compiling htop for Windows from Linux
+
+An intermediate build using Cygwin/MSYS is being worked on by @ookiineko as of 2023-09-28 (see #1097)
```

Comment:
while you can use them as a starting point for the native port, I think Cygwin/Msys ports are NOT intermediate builds, they should be considered as standalone unix-like platforms. See: https://github.com/htop-dev/htop/issues/1097#issuecomment-1740194685

Reply:
I think their existence should be mentioned in this document.  Not sure on the best wording.

I don't agree with your view of the platforms or usefulness them beyond a native port, but that have nothing to do with this PR.